docs/en/sql-reference/functions/encoding-functions.md

```diff
@@ -151,10 +149,62 @@ Result:
 └──────────────────┘
 ```
 
-## unhex(str) {#unhexstr}
+## unhex {#unhexstr}
+
+Performs the opposite operation of [hex](#hex). It interprets each pair of hexadecimal digits (in the argument) as a number and converts it to the byte represented by the number. The return value is a binary string.
```

Comment:
The term `binary string` usually refers to a string, that consists of 0-s and 1-s. But also it is used to define blobs in  SQL (types `BINARY` and `VARBINARY` in MySQL. Maybe add a clarification, that it is related to blob in that case.